graylog2-web-interface/src/views/components/visualizations/worldmap/MapVisualization.jsx

```diff
@@ -94,8 +96,14 @@ const MapVisualization = createReactClass({
             {markerKeys}
             <dt>Coordinates:</dt>
             <dd>{coordinates}</dd>
-            <dt>Value:</dt>
-            <dd>{value}</dd>
+            {value
+              && (
+              <React.Fragment>
+                <dt>Value:</dt>
+                <dd>{value}</dd>
+              </React.Fragment>
+              )
+            }
```

Comment:
Is it fine to display the value conditionally or is there any case where we want to display a value of 0?

Reply:
No, that is fine. We actually have no value here.